### PR TITLE
Pp 863 card types used available to partners

### PR DESCRIPTION
**Pull request by SandorArpa**
The used card brand, eg visa, master-card, etc is available for search as card_brand. The API returns the Label of the card brand, eg Visa, Mastercard, etc with charges status greater than 'CAPTURE SUBMITTED'

### CODE REVIEW

**Comment by ckalista** [comment]
rebuild

**Comment by ckalista** [comment]
rebuild

**Comment by ckalista** [comment]
build

**Comment by ckalista** [comment]
rebuild

File: src/main/java/uk/gov/pay/api/model/ChargeFromResponse.java [hunk 1]
```diff
@@ -20,6 +20,9 @@
     @JsonProperty("payment_provider")
     private String paymentProvider;
 
+    @JsonProperty("card_brand")
+    private String cardLabel;
```
Comment (rahul-vinayak) [inline src/main/java/uk/gov/pay/api/model/ChargeFromResponse.java]: if (brand != label) 
        "pick a better property"

File: src/main/java/uk/gov/pay/api/model/PaymentForSearchResult.java [hunk 2]
```diff
@@ -40,6 +40,7 @@ public static PaymentForSearchResult valueOf(
                 paymentResult.getReference(),
                 paymentResult.getEmail(),
                 paymentResult.getPaymentProvider(),
+                paymentResult.getCardLabel(),
```
Comment (rahul-vinayak) [inline src/main/java/uk/gov/pay/api/model/PaymentForSearchResult.java]: are label and brand used for same thing?

**Comment by rahul-vinayak** [comment]
since the email and reference searches have been made case-insensitive while searching, would be good to make sure the card type is case-insensitive as well, make sure connector end point would accept both "Visa" as well as "visa". So would be be good to write a test for that, and the required change is in the connector layer makes it case-insensitive, like the ref and email.

File: src/main/java/uk/gov/pay/api/model/ChargeFromResponse.java (hunk 1 above)
Comment (SandorArpa) [inline src/main/java/uk/gov/pay/api/model/ChargeFromResponse.java]: I have addressed the issue. Renamed variable accordingly.

**Comment by SandorArpa** [comment]
rebuild

File: src/main/java/uk/gov/pay/api/model/PaymentForSearchResult.java (hunk 2 above)
Comment (SandorArpa) [inline src/main/java/uk/gov/pay/api/model/PaymentForSearchResult.java]: I have addressed the issue. Renamed methods accordingly.

**Comment by SandorArpa** [comment]
rebuild

**Comment by rahul-vinayak** [comment]
👍

**Pull request merged**